include/z64dma.h

```diff
@@ -4,6 +4,18 @@
 #include "ultra64.h"
 #include "alignment.h"
 
+typedef struct {
+    /* 0x00 */ uintptr_t vromStart;
+    /* 0x04 */ uintptr_t vromEnd;
+} RomFile; // size = 0x8
```

Comment:
Do wonder if in all cases if we should call the variable `rom` and then change the variable names to just `start` and `end`
So in context it would look like `thing->rom.start`

I guess the way it is right now is fine, especially if you really value the specificity of "vrom", but it does feel redundant to say in the variable name now. More thoughts needed

Reply:
FWIW `RomFile rom` doesn't seem right to me, I don't think I would call a start and end address a "rom" in conversation, it's more like a section of a rom

Reply:
the type is still RomFile which tells you what it is, but `rom.start` reads exactly as you would use it in conversation "the rom start address"
but yeah idk

Reply:
I don't find there's anything superfluous in `file.vromStart`/End, and I think it's better to "abstract" that a file is located by vrom by naming the fields after that rather than the file member

Reply:
I had thought about doing something like `RomFile vrom` with `vrom.start` and `vrom.end`. Biggest hesitation for me was the few things that just use `RomFile` directly. For example the ObjectTable would have accesses like `gObjectTable[i].end - gObjectTable[i].start`, which doesn't seem right. I would want to wrap it in another struct to have `vrom` as the struct member. Not sure if that was worth it.

Reply:
yeah does not seem worth it. its fine as is